src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -1,209 +1,84 @@
-
-  // Insets
-  --#{$toolbar}--m-page-insets--inset: var(--#{$pf-global}--spacer--md); // make this the default inset at breaking change
-  --#{$toolbar}--m-page-insets--xl--inset: var(--#{$pf-global}--spacer--lg); // make this the default inset at breaking change
-
-  // Expandable content
-  --#{$toolbar}__expandable-content--Display: grid;
-  --#{$toolbar}__expandable-content--PaddingTop: 0;
-  --#{$toolbar}__expandable-content--PaddingRight: var(--#{$toolbar}__content--PaddingRight);
-  --#{$toolbar}__expandable-content--PaddingBottom: var(--#{$pf-global}--spacer--md);
-  --#{$toolbar}__expandable-content--PaddingLeft: var(--#{$toolbar}__content--PaddingLeft);
-  --#{$toolbar}__expandable-content--lg--PaddingRight: 0;
-  --#{$toolbar}__expandable-content--lg--PaddingBottom: 0;
-  --#{$toolbar}__expandable-content--lg--PaddingLeft: 0;
-  --#{$toolbar}__expandable-content--ZIndex: var(--#{$pf-global}--ZIndex--sm);
-  --#{$toolbar}__expandable-content--BoxShadow: var(--#{$pf-global}--BoxShadow--md-bottom);
-  --#{$toolbar}__expandable-content--BackgroundColor: var(--#{$toolbar}--BackgroundColor);
-
-  // Expanded
-  --#{$toolbar}__expandable-content--m-expanded--GridRowGap: var(--#{$pf-global}--gutter--md);
-
-  // Chip container
-  --#{$toolbar}__group--m-chip-container--MarginTop: calc(var(--#{$pf-global}--spacer--md) * -1);
-  --#{$toolbar}__group--m-chip-container__item--MarginTop: var(--#{$pf-global}--spacer--md);
-
-  // Base spacer - shared value
-  --#{$toolbar}--spacer--base: var(--#{$pf-global}--spacer--md);
-
-  // Spacer values
-  --#{$toolbar}__item--spacer: var(--#{$toolbar}--spacer--base);
-  --#{$toolbar}__item--Width: auto;
-  --#{$toolbar}__group--spacer: var(--#{$toolbar}--spacer--base);
-
-  // Toggle group
-  --#{$toolbar}__group--m-toggle-group--spacer: var(--#{$pf-global}--spacer--sm);
-
-  // update
-  --#{$toolbar}__group--m-toggle-group--m-show--spacer: var(--#{$toolbar}__group--spacer);
-
-  // Icon button group
-  --#{$toolbar}__group--m-icon-button-group--spacer: var(--#{$toolbar}__group--spacer);
-  --#{$toolbar}__group--m-icon-button-group--space-items: 0;
-
-  // Button group
-  --#{$toolbar}__group--m-button-group--spacer: var(--#{$toolbar}__group--spacer);
-  --#{$toolbar}__group--m-button-group--space-items: var(--#{$pf-global}--spacer--sm);
+$pf-v5-c-toolbar--spacer-map: build-spacer-map();
+$pf-v5-c-toolbar--inset-map: build-spacer-map();
+$pf-v5-c-toolbar--align-items--alignments: ('stretch', 'baseline', 'start', 'center', 'end');
+$pf-v5-c-toolbar--breakpoint-map: build-breakpoint-map('base');
+$pf-v5--include-toolbar-breakpoints: true !default;
+
+@if $pf-v5--include-toolbar-breakpoints {
+  $pf-v5-c-toolbar--breakpoint-map: build-breakpoint-map('base', 'sm', 'md', 'lg', 'xl', '2xl');
+}
 
-  // Filter group
-  --#{$toolbar}__group--m-filter-group--spacer: var(--#{$toolbar}__group--spacer);
-  --#{$toolbar}__group--m-filter-group--space-items: 0;
+:root,
+[data-theme="#{$toolbar}"] {
```

Comment:
Is this meant to be here, or should be the `:where` pseudo?

Reply:
This is intentional in that, `:where(.component-root)` adds the component scope to each instance which overrides any variable updates provided by a parent component. @mcoker, @srambach and I talked about this last week and concluded that `[data-theme="#{$component}"]`, where components could be potentially nested, adds an optional, non-root component namespaced "reset" option separate from the component selector. This allows us to `reset` any component/child component when needed.